Cargo.toml

```diff
@@ -20,6 +20,7 @@ features = ["winit"]
 [dependencies]
 stretch = "0.2"
 twox-hash = "1.5"
+log = "0.4.8"
```

Comment:
We are not using the dependency for anything yet... We should probably add it once we need it.